tests/rptest/tests/franz_go_verifiable_test.py

```diff
@@ -91,54 +93,37 @@ def test_with_all_type_of_loads(self):
         assert self._rand_consumer.consumer_status.total_reads == self.RANDOM_READ_COUNT * self.RANDOM_READ_PARALLEL
 
 
+# archival - [fiber63 kafka/topic-vmumdxkmeg/33] - ntp_archiver_service.cc:96 - upload loop error: seastar::timed_out_error (timedout)
+KGO_LOG_ALLOW_LIST = [
+    re.compile(
+        'archival - .*upload loop error: seastar::timed_out_error (timedout)')
+]
+
+KGO_RESTART_LOG_ALLOW_LIST = KGO_LOG_ALLOW_LIST + RESTART_LOG_ALLOW_LIST
+
+
 class FranzGoVerifiableWithSiTest(FranzGoVerifiableBase):
     MSG_SIZE = 1000000
     PRODUCE_COUNT = 20000
     RANDOM_READ_COUNT = 1000
     RANDOM_READ_PARALLEL = 20
 
-    segment_size = 5 * 1000000
-
     topics = (TopicSpec(partition_count=100, replication_factor=3), )
 
     def __init__(self, ctx):
-        si_settings = SISettings(
-            log_segment_size=FranzGoVerifiableWithSiTest.segment_size,
-            cloud_storage_cache_size=5 *
-            FranzGoVerifiableWithSiTest.segment_size)
-
-        super(FranzGoVerifiableWithSiTest, self).__init__(
-            test_context=ctx,
-            num_brokers=3,
-            extra_rp_conf={
-                # Disable prometheus metrics, because we are doing lots
-                # of restarts with lots of partitions, and current high
-                # metric counts make that sometimes cause reactor stalls
-                # during shutdown on debug builds.
-                'disable_metrics': True,
-
-                # We will run relatively large number of partitions
-                # and want it to work with slow debug builds and
-                # on noisy developer workstations: relax the raft
-                # intervals
-                'election_timeout_ms': 5000,
-                'raft_heartbeat_interval_ms': 500,
```

Comment:
According to the comments, these config options were enabled for debug builds. However, the tests are skipped in debug mode so I removed them.